common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -96,6 +96,16 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-acquisitions-epic-native-vs-dfp-v2",
+    "See if there is any difference in annualised value between serving the Epic natively vs DFP",
+    owners = Seq(Owner.withGithub("Mullefa")),
+    safeState = On,
```

Comment:
Do we need to use safestate on? Most of our switches are off by default.

Reply:
I've opted for safestate on since it prevents the issue of the test not getting displayed because we have forgotten to turn the switch on - something that has arisen in the past!

I don't mind changing it but would be keen to hear your concern?